### PR TITLE
[DocsArchive-1.0.23045.1] Archive binary serialization articles

### PR DESCRIPTION
**Pull request by gewarren**
***Please don't merge this PR until target archive repo PR https://github.com/dotnet/docs-archive-pr/pull/33 is merged into live branch.*** 
 Auto archive content to https://github.com/gewarren/docs-archive-pr.git

Contributes to #32568 

---

#### Internal previews

<details><summary><strong>Toggle expand/collapse</strong></summary><br/>

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/architecture/maui/mvvm-community-toolkit-features.md](https://github.com/dotnet/docs/blob/db98db8b5a5aea22399ea0898af4fb74c9896be4/docs/architecture/maui/mvvm-community-toolkit-features.md) | [docs/architecture/maui/mvvm-community-toolkit-features](https://review.learn.microsoft.com/en-us/dotnet/architecture/maui/mvvm-community-toolkit-features?branch=pr-en-us-34847) |
| [docs/core/compatibility/unsupported-apis.md](https://github.com/dotnet/docs/blob/db98db8b5a5aea22399ea0898af4fb74c9896be4/docs/core/compatibility/unsupported-apis.md) | [docs/core/compatibility/unsupported-apis](https://review.learn.microsoft.com/en-us/dotnet/core/compatibility/unsupported-apis?branch=pr-en-us-34847) |
| [docs/csharp/programming-guide/concepts/linq/filtering-data.md](https://github.com/dotnet/docs/blob/db98db8b5a5aea22399ea0898af4fb74c9896be4/docs/csharp/programming-guide/concepts/linq/filtering-data.md) | [docs/csharp/programming-guide/concepts/linq/filtering-data](https://review.learn.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/linq/filtering-data?branch=pr-en-us-34847) |
| [docs/csharp/programming-guide/concepts/linq/linq-to-objects.md](https://github.com/dotnet/docs/blob/db98db8b5a5aea22399ea0898af4fb74c9896be4/docs/csharp/programming-guide/concepts/linq/linq-to-objects.md) | [docs/csharp/programming-guide/concepts/linq/linq-to-objects](https://review.learn.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/linq/linq-to-objects?branch=pr-en-us-34847) |
| [docs/framework/data/adonet/sql/linq/sql-clr-type-mapping.md](https://github.com/dotnet/docs/blob/db98db8b5a5aea22399ea0898af4fb74c9896be4/docs/framework/data/adonet/sql/linq/sql-clr-type-mapping.md) | [docs/framework/data/adonet/sql/linq/sql-clr-type-mapping](https://review.learn.microsoft.com/en-us/dotnet/framework/data/adonet/sql/linq/sql-clr-type-mapping?branch=pr-en-us-34847) |
| [docs/framework/wcf/feature-details/serialization-and-deserialization.md](https://github.com/dotnet/docs/blob/db98db8b5a5aea22399ea0898af4fb74c9896be4/docs/framework/wcf/feature-details/serialization-and-deserialization.md) | [docs/framework/wcf/feature-details/serialization-and-deserialization](https://review.learn.microsoft.com/en-us/dotnet/framework/wcf/feature-details/serialization-and-deserialization?branch=pr-en-us-34847) |
| [docs/framework/wcf/feature-details/version-tolerant-serialization-callbacks.md](https://github.com/dotnet/docs/blob/db98db8b5a5aea22399ea0898af4fb74c9896be4/docs/framework/wcf/feature-details/version-tolerant-serialization-callbacks.md) | [docs/framework/wcf/feature-details/version-tolerant-serialization-callbacks](https://review.learn.microsoft.com/en-us/dotnet/framework/wcf/feature-details/version-tolerant-serialization-callbacks?branch=pr-en-us-34847) |
| [docs/fundamentals/code-analysis/quality-rules/ca1801.md](https://github.com/dotnet/docs/blob/db98db8b5a5aea22399ea0898af4fb74c9896be4/docs/fundamentals/code-analysis/quality-rules/ca1801.md) | [docs/fundamentals/code-analysis/quality-rules/ca1801](https://review.learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1801?branch=pr-en-us-34847) |
| [docs/orleans/host/configuration-guide/serialization.md](https://github.com/dotnet/docs/blob/db98db8b5a5aea22399ea0898af4fb74c9896be4/docs/orleans/host/configuration-guide/serialization.md) | [docs/orleans/host/configuration-guide/serialization](https://review.learn.microsoft.com/en-us/dotnet/orleans/host/configuration-guide/serialization?branch=pr-en-us-34847) |
| [docs/standard/serialization/binaryformatter-security-guide.md](https://github.com/dotnet/docs/blob/db98db8b5a5aea22399ea0898af4fb74c9896be4/docs/standard/serialization/binaryformatter-security-guide.md) | [docs/standard/serialization/binaryformatter-security-guide](https://review.learn.microsoft.com/en-us/dotnet/standard/serialization/binaryformatter-security-guide?branch=pr-en-us-34847) |
| [docs/standard/serialization/how-to-chunk-serialized-data.md](https://github.com/dotnet/docs/blob/db98db8b5a5aea22399ea0898af4fb74c9896be4/docs/standard/serialization/how-to-chunk-serialized-data.md) | [docs/standard/serialization/how-to-chunk-serialized-data](https://review.learn.microsoft.com/en-us/dotnet/standard/serialization/how-to-chunk-serialized-data?branch=pr-en-us-34847) |
| [docs/standard/serialization/index.md](https://github.com/dotnet/docs/blob/db98db8b5a5aea22399ea0898af4fb74c9896be4/docs/standard/serialization/index.md) | [docs/standard/serialization/index](https://review.learn.microsoft.com/en-us/dotnet/standard/serialization/index?branch=pr-en-us-34847) |
| [docs/standard/serialization/introducing-xml-serialization.md](https://github.com/dotnet/docs/blob/db98db8b5a5aea22399ea0898af4fb74c9896be4/docs/standard/serialization/introducing-xml-serialization.md) | [docs/standard/serialization/introducing-xml-serialization](https://review.learn.microsoft.com/en-us/dotnet/standard/serialization/introducing-xml-serialization?branch=pr-en-us-34847) |
| [docs/standard/serialization/system-text-json/how-to.md](https://github.com/dotnet/docs/blob/db98db8b5a5aea22399ea0898af4fb74c9896be4/docs/standard/serialization/system-text-json/how-to.md) | [docs/standard/serialization/system-text-json/how-to](https://review.learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/how-to?branch=pr-en-us-34847) |
| [docs/standard/serialization/xml-and-soap-serialization.md](https://github.com/dotnet/docs/blob/db98db8b5a5aea22399ea0898af4fb74c9896be4/docs/standard/serialization/xml-and-soap-serialization.md) | [docs/standard/serialization/xml-and-soap-serialization](https://review.learn.microsoft.com/en-us/dotnet/standard/serialization/xml-and-soap-serialization?branch=pr-en-us-34847) |
| [docs/whats-new/dotnet-7-docs.md](https://github.com/dotnet/docs/blob/db98db8b5a5aea22399ea0898af4fb74c9896be4/docs/whats-new/dotnet-7-docs.md) | [docs/whats-new/dotnet-7-docs](https://review.learn.microsoft.com/en-us/dotnet/whats-new/dotnet-7-docs?branch=pr-en-us-34847) |

</details>

### CODE REVIEW

File: docs/fundamentals/toc.yml [hunk 1]
```diff
@@ -460,7 +460,7 @@ items:
         href: ../standard/base-types/divide-up-strings.md
       - name: Use the StringBuilder class
         href: ../standard/base-types/stringbuilder.md
-      - name: "How to: Perform basic string manipulations"
+      - name: 'How to: Perform basic string manipulations'
```
Comment (tdykstra) [inline docs/fundamentals/toc.yml:463]: What's the advantage of single quotes over double quotes here?

File: docs/fundamentals/toc.yml [hunk 2]
```diff
@@ -92,69 +92,69 @@ items:
     - name: Use Visual Studio
       items:
       - name: Create a console app
-        href: ../core/tutorials/with-visual-studio.md
         displayName: tutorials, visual studio, vs
+        href: ../core/tutorials/with-visual-studio.md
```
Comment (tdykstra) [inline docs/fundamentals/toc.yml:96]: What's the reason for ordering name-displayname-href instead of name-href-displayname?

**Review by tdykstra — APPROVED** [review]
LGTM, just a couple of questions.

File: docs/fundamentals/toc.yml (hunk 2 above)
Comment (gewarren) [reply docs/fundamentals/toc.yml:96]: The archive tool made all these changes automatically. I'll revert them.

File: docs/fundamentals/toc.yml (hunk 1 above)
Comment (gewarren) [reply docs/fundamentals/toc.yml:463]: Archive tool again...I'll revert.

**Comment by gewarren** [comment]
Pending approval from @GrabYourPitchforks.

**Pull request merged**